.github/workflows/lighthouse_accessibility.yml

```diff
@@ -0,0 +1,19 @@
+name: Lighthouse CI accessibility tests
+on: [pull_request]
```

Comment:
Is this supposed to run only on pull requests? Won't we want to run them on pushes to develop or release branches as well?

Reply:
I think we want it to work on pushes to develop and release branches as well. Let me make sure, @jameesjohn @vojtechjelinek  is there any reason why we can't?

Reply:
I think it's okay to run it on pushes to release and develop branches

Reply:
Waiting on this being resolved before approval for codeowners.

Reply:
LGTM now!